ts/Extensions/Breadcrumbs.ts

```diff
@@ -0,0 +1,1180 @@
+            breadcrumbsList.forEach(function (el): void {
+                const button = el[2],
+                    separator = el[3];
+
+                // Remove SVG elements fromt the DOM.
+                button && button.destroy();
+                separator && separator.destroy();
+            });
+            // Clear the breadcrums list array.
+            breadcrumbsList.length = 0;
+            this.breadcrumbsGroup.destroy();
+            this.breadcrumbsGroup = void 0 as any;
+        } else if (
+            chart.drillUpButton
+        ) {
+            breadcrumbsList.length = 0;
+            chart.drillUpButton.destroy();
+            chart.drillUpButton = void 0;
+        }
+        chart.getMargins();
+    }
+
+    /**
+     * Default button text formatter.
+     *
+     * @requires  modules/breadcrumbs
+     *
+     * @function Highcharts.Breadcrumbs#getButtonText
+     * @param {Highcharts.Breadcrumbs} this
+     *        Breadcrumbs class.
+     * @param {Highcharts.Breadcrumbs} breadcrumb
+     *        Breadcrumb.
+     * @return {string}
+     *         Formatted text.
+     */
+    public getButtonText(this: Breadcrumbs, breadcrumb: Array<number|null|Point|SeriesType|SVGAElement>): string {
+        const breadcrumbs = this,
+            chart = breadcrumbs.chart,
+            breadcrumbsOptions = breadcrumbs.options,
+            lang = chart.options.lang,
+            textFormat = pick(breadcrumbsOptions.format,
+                breadcrumbsOptions.showFullPath ? '{level.name}' : '← {level.name}'
+            );
+        let returnText = breadcrumbsOptions.formatter && breadcrumbsOptions.formatter(breadcrumb, breadcrumbs) ||
+                format(textFormat, { level: (breadcrumb[1] as Point) }, chart) || '';
+
+        if (returnText === '← ' && lang && lang.mainBreadcrumb) {
+            returnText = !breadcrumbsOptions.showFullPath ? '← ' + lang.mainBreadcrumb : lang.mainBreadcrumb;
+        }
+        return returnText;
+    }
+
+    /**
+     * Perform the drillUp action.
+     *
+     * @requires  modules/breadcrumbs
+     *
+     * @function Highcharts.Breadcrumbs#jumpUp
+     * @param {Highcharts.Breadcrumbs} this
+     *        Breadcrumbs class.
+     */
+    public jumpUp(this: Breadcrumbs): void {
+        const breadcrumbs = this,
+            chart = breadcrumbs.chart;
+
+        if (this.series && this.series.drillUp) {
+            this.series.drillUp();
+        } else {
+            chart.drillUp();
+            breadcrumbs.calculateLevel();
+        }
+    }
+
+    /**
+     * Performs multiple drillups based on the provided number.
+     *
+     * @requires  modules/breadcrumbs
+     *
+     * @function Highcharts.Breadcrumbs#jumpBy
+     * @param {Highcharts.Breadcrumbs} this
+     *        Breadcrumbs class.
+     * @param {Highcharts.Breadcrumbs} jumpsAmount
+     *        A number of drillups that needs to be performed.
```

Comment:
I think I'm missing something here. `jumpsAmount` suppose to be the number of drillups, yet later in the function `const jumpsNumber` is calculated to get the number of drillups.

Reply:
Perhaps the method name should be `jumpTo(level)`?

Reply:
yes, it should be `jumpTo`, going to the level related to clicked breadcrumb